Update 3 Controlled Mode.md (#1396)
DevExpress/devextreme-documentation: concepts/50 React Components/20 State Management/3 Controlled Mode.md

In <a href="https://reactjs.org/docs/forms.html#controlled-components" target="_blank">controlled mode</a>, the parent React component updates a DevExtreme component's state. It should pass new values to the DevExtreme component via attributes, as with the `value` attribute of the `TextBox` component in the following code.

> DevExtreme components raise events that you should handle to update the parent component's state (the `handleValueChange` function in the code below). These events are raised only when a user interacts with the component, not when you update an attribute value programmatically. A handler gets a new value as a parameter.
> 
> In general case there is a naming convention: to handle `optionName` change pass a handler to `onOptionNameChange` property.

Reviewer: ```suggestion
DevExtreme components raise events that you should handle to update the parent component's state. These events are raised only when a user interacts with the component, not when you update an attribute value programmatically. Each event name is based on the attribute name. For example, the event for the `value` attribute is `onValueChange`. An event handler accepts a new attribute value as a parameter:
```
